easybuild/easyconfigs/s/snpEff/snpEff-4.3k-Java-1.7.0_80.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PackedBinary'
+
+name = 'snpEff'
+version = '4.3k'
+
+homepage = 'http://sourceforge.net/projects/snpeff/'
+description = """SnpEff is a variant annotation and effect prediction tool. 
+ It annotates and predicts the effects of genetic variants (such as amino acid changes)."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+java = 'Java'
+javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
```

Comment:
@danielnavarrogomez We have a better way of doing this now for a while, using the `%(javaver)s` template, see also http://easybuild.readthedocs.io/en/latest/version-specific/easyconfig_templates.html#avail-easyconfig-templates

I made this changed consistently in https://github.com/danielnavarrogomez/easybuild-easyconfigs/pull/1; if you merge that pull request, this one will be updated accordingly.

Reply:
@danielnavarrogomez ping?